ecmascript/parser/src/lib.rs

```diff
@@ -295,6 +302,10 @@ pub struct TsConfig {
     /// Stage 3.
     #[serde(default)]
     pub import_assertions: bool,
+
+    #[serde(default)]
+    #[serde(rename = "staticBlocks")]
+    pub static_blocks: bool,
```

Comment:
Is static block optional for typescript?
If `tsc` allows it by default, `swc` should allow it without a configuration, and if `tsc` does not support it, `swc` should not allow it for typescript files.

Reply:
Since TS 4.4, it is not optional, fully supported. I'll fix it.

Reply:
Thanks!